governance/chair-proposal-process.md

```diff
@@ -0,0 +1,148 @@
+# Environmental Sustainability Leadership Team
+
+The leadership team of the WG are servant leads. They are an enabler for the group and support the creation and success of the different activities. They are the driver for the group for not losing the higher targets out of sight and taking decisions when no consensus can be found.
+
+## Team Structure
+
+The leadership team is conducted of:
+* Three Co-Chairs
+* Three Tech Leads
+
+As WG we strive to be a gender, company and geolocation, diverse group. 
+Therefore, the whole leadership team is not allowed to have more than two people from the same organization. This also includes companies that belong through majority ownership to another company. 
```

Comment:
As you mentioned in the last meeting we should try to do it better. 

Let's say we have two chairs from Company A and two tech leads from Company B, where Company A owns a significant part of Company B, this should be considered the same organization. This shouldn't be allowed, because it doesn't guarantee diversity regards company representation. Or in another word, the two chairs and the two tech leads can always get a majority in a vote.

Reply:
Why this is not allowed? Why doesn't it guarantee diversity, and in which way? It is not only discriminative to evaluate Open Source contributors based on their employments, it is full of fallacy as well. What do you do when people change jobs, go through company acquisitions, or conduct consulting projects?

Successful Open Source communities guarantee fairness and diversity through transparency and openness, not through discriminative rules.

Reply:
@mkorbi  it really depends on the size of the company.  I can see a small company having little diversity, but companies that work across Geos have much diversity, many different teams, and many of these have not only different goals, but different cultures.

Reply:
As an alternative solution with precedence, this is addressed by organization voting caps in the governance of projects such as [Thanos](https://github.com/thanos-io/thanos/blob/main/docs/governance.md#governance-1), [CNI](https://www.cni.dev/docs/governance/), and [Cortex](https://github.com/cortexproject/cortex/blob/master/GOVERNANCE.md).